docs/configuration.asciidoc

```diff
@@ -16,7 +16,7 @@ WARNING: Due to the complex nature of the configuration, the config object you p
 [[config-options]]
 === Config options
 [horizontal]
-`host or hosts`[[config-hosts]]::
+[[config-hosts]]`host or hosts`::
```

Comment:
This file is generated from https://github.com/elastic/elasticsearch-js/blob/legacy-doc/scripts/generate/templates/configuration_docs.tmpl, so we'll need to make sure and update that template and run `node scripts/generate` to update this file.

Reply:
Thanks! I'll take care of this.